### ISSUE TITLE
Sufficient Model.query documentation

### ISSUE THREAD

**Issue by Boycce** [balderdashy/sails-docs#724]
Hello, I cannot seem to find any sufficient documentation for `Model.query`. It would be nice to have examples of all the basic sql operations (select, insert, update, delete) on the page reference below.
http://sailsjs.org/documentation/reference/waterline-orm/models/query

This issue request arose from when trying to perform a prepared `insert` query:
``` Javascript
var text = 'INSERT INTO posts (name, slug) VALUES ($1, $2), ($1, $2)';
var values = ['Post 1', 'post1', 'Post 2', 'post2'];
Post.query({text: text, values: values}, cb);
```
and receiving a `ER_EMPTY_QUERY` error:

> added { [Error: ER_EMPTY_QUERY: Query was empty]
>   code: 'ER_EMPTY_QUERY',
>   errno: 1065,
>   sqlState: '42000',
>   index: 0,
>   originalError: [Circular] } undefined

**Comment by sailsbot** [comment]
@Boycce: Hello, I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message and simply close the issue if possible.  On the other hand, if you are still waiting on a patch, please post a comment to keep the thread alive (with any new information you can provide).



If no further activity occurs on this thread within the next 3 days, the issue will automatically be closed.

Thanks so much for your help!

**Comment by jcowley** [comment]
@Boycce: Yeah, I'm getting the same error using the example prepared statement query almost verbatim. Apparently the docs are incorrect.

But thanks to that helpful bot, I guess the issue is closed and we don't need to worry about it, eh?

**Comment by jcowley** [comment]
For anyone else that's trying to understand how to get the prepared statement example to work, the documentation appears to be incorrect (at least for MySQL). 

Corrected version of the example in the docs:
```
Pet.query(
  'SELECT pet.name FROM pet WHERE pet.name = $1', 
  [ "dog" ], 
  function(err, results) {
    if (err) return res.serverError(err);
    return res.ok(results.rows);
  }
);
```
I would update the docs and submit a PR, but I suspect there are differences when using Postgres and I don't have the time right now to set up a test for that. Maybe the example in the docs is based on Postgres?
